presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -256,6 +256,17 @@ class SystemConfig : public ConfigBase {
   /// this config only applies if the memory arbitration has been enabled.
   static constexpr std::string_view kQueryMemoryGb{"query-memory-gb"};
 
+  /// Specifies the amount of query memory capacity reserved to ensure that a
+  /// query has minimal memory capacity to run. This capacity should be less
+  /// than 'query-memory-gb'. A query's minimal memory capacity is specified by
```

Comment:
The relation between 'query-reserved-memory-gb' and 'memory-pool-reserved-capacity' is a bit difficult to follow. Can you elaborate ?

Reply:
Updated.